### ISSUE TITLE
fasta file upload error

### ISSUE THREAD

**Issue by AnunayS** [nanoporetech/tombo#347]
I tried several commands after the reference file path in the resquiggle command but every time it is showing the following error.
![Untitled](https://user-images.githubusercontent.com/85170013/120891748-d38a1c00-c627-11eb-9c77-cffbe3db27d4.png)

However the minimap2 is working fine. I tried running basic command given on the minimap2 github page, they are working fine.

Please let me know if any one can help me with the issue.

Thankyou.

**Comment by Chris-Kimmel** [comment]
Hi AnunayS. Are you totally sure that your FASTA filepath is correct?

**Comment by AnunayS** [comment]
Yes the fasta file path is correct. I am just not sure whether the command I am writing is correct or not because I am finding several different commands for the same.

**Comment by Chris-Kimmel** [comment]
@AnunayS Have you tried removing the leading slashes from the filepaths?

**Comment by AnunayS** [comment]
I tried that too but still the same error.
It will be kind of you if you can give me a command that has actually worked for you so that I can try it out.
That way I will be sure that the command is correct but there can be some installation problem with tombo or minimap2.

**Comment by marcus1487** [comment]
You could run `python -c 'import mappy; aligner = mappy.Aligner("/tombotest/fast5/msmeg.fasta")'` to help diagnose the underlying issue here.
